VideoWeb/VideoWeb/ClientApp/src/app/shared/cookie-banner/cookie-banner.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, EventEmitter, Output } from '@angular/core';
+import { cookies } from '../cookies.constants';
+
+@Component({
+    selector: 'app-cookie-banner',
+    templateUrl: './cookie-banner.component.html',
+    styleUrls: ['./cookie-banner.component.css']
+})
+export class CookieBannerComponent {
+    @Output() isBannerVisibleChange = new EventEmitter<boolean>();
```

Comment:
the variable name and the action don't line up here. Could you label this as `cookieAnswered` or `cookieAccepted`

Reply:
done